docs/shared/pods-setup-codegen-panel.mdx

```diff
@@ -12,10 +12,11 @@ The Codegen CLI is located in your project at `Pods/Apollo/apollo-ios-cli`
 The Codegen CLI uses a JSON file to configure the code generation engine. Use the Codegen CLI `init` command to create this file with default values. From the directory of your `Podfile`, run:
 
 ```bash
-./Pods/Apollo/apollo-ios-cli init --schema-name ${MySchemaName}
+./Pods/Apollo/apollo-ios-cli init --schema-name ${MySchemaName} --module-type other
 ```
 
-`${MySchemaName}` should be the name you want for the namespace of your generated schema files.
+* `${MySchemaName}` should be the name you want for the namespace of your generated schema files.
+* `other` is the module type to use when your dependency manager for Apollo iOS is CocoaPods.
```

Comment:
I think I have a problem with this. Let's talk about it more.

I don't think that telling them it's a no-brainer and just to use `other` if they are using Cocoapods is accurate. 

`other` is used if you want to package your generated schema as a separate module and distribute that as a Pod. But what we've found is that _most_ users actually just want to embed the generated models right into their application target. They are just including `Apollo` via Cocoapods. 

Even though we don't believe that is the best choice, it does seem to be the default expectation of users. So I think `embeddedInTarget` is still the default module type people would want to use. Either way, we should be very clear in explaining to people that this is a choice to be made and they should understand what the implications of each possible choice is. That way we aren't making `embeddedInTarget` the path of least resistance, but we also aren't confusing people who assumed that it would work that way, when their generated files don't link to their project and build properly.

Reply:
This is in a CocoaPods focused panel, which is why it says to use `other` - it's contextual.

If they added Apollo to their project via CocoaPods the CLI is built with the `COCOAPODS` flag and will therefore always add `cocoapodsCompatibleImportStatements=true` to the initialization file. This happens without them knowing and if they used anything other than `other` the generated code wouldn't build because the `import` statements would be incorrect.

There is nuance and edge cases but the overwhelming majority of users that use CocoaPods should be using `other` as the module type here.